<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameUI</class>
 <widget class="QWidget" name="GameUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>594</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1663</width>
    <height>1112</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-290</x>
     <y>-130</y>
     <width>1421</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background-image: url(:/gamepage/table/Game-Ground-Photo.JPG);
    background-repeat: no-repeat;
    background-position: center;
}</string>
   </property>
   <widget class="QPushButton" name="requestedCardButton5_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectedCardButton5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>600</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectedCardButton4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>600</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectedCardButton2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectedCardButton1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectedCardButton3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>600</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>140</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resumeButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 0, 0);</string>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerlabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 127);
alternate-background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="requestedCardButton4_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="chatFrame">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 125, 125);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="infolabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>610</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
